README.md

```diff
@@ -1,66 +1,128 @@
+   
+   IMPORTANT: To prevent inadvertent (and possibly incompatible) updates, ensure that there are no network cables connected to the Pi. 
+   
+   On first boot, follow the instructions to set your locale settings, connect to a WiFi network and update the kernel.
 
 2. Ensure running kernel version matches headers kernel headers package. A typical system requires the following `--reinstall` command:
 
-   ```sudo apt-get install --reinstall raspberrypi-bootloader raspberrypi-kernel```
+   ```
+   sudo apt-get update
+   sudo apt-get install --reinstall raspberrypi-bootloader raspberrypi-kernel
+   ```
 
    followed by a reboot.
 
-3. Open a terminal on the Raspberry Pi and clone this repository:  
-`cd ~; git clone https://github.com/xmos/vocalfusion-avs-setup`
+3. Clone the vocalfusion-avs-setup repository:
+
+   ```git clone https://github.com/xmos/vocalfusion-avs-setup```
+
+4. Close any other application such as browsers to avoid the Raspberry Pi freezing during the AVS SDK installation.
+
+5. Register an AVS product, create a security profile and save a *config.json* file by following https://developer.amazon.com/en-US/docs/alexa/alexa-voice-service/register-a-product.html
+
+6. Copy the *config.json* into the directory `vocalfusion-avs-setup`
+
+7. Run the installation script by entering:
+
+   ``` cd vocalfusion-avs-setup```
+
+   And then either
+
+   ```./auto_install.sh xvf3510```
 
-4. Create a new Alexa device by following: https://github.com/alexa/alexa-avs-sample-app/wiki/Create-Security-Profile  
-(Note: the *Allowed Origins* and *Allowed Return URLs* should use **http**, not https.)  
+   for the xCORE VocalFusion XVF3510,
 
-5. Run the installation script: `source ~/vocalfusion-avs-setup/auto_install.sh`    
-Read and accept the AVS Device SDK license agreement.
+   ```./auto_install.sh xvf3500```
 
-6. You will be prompted enter your Alexa device details and asked whether you want the Sample App to run automatically when the Raspberry Pi boots.
+   for the xCORE VocalFusion XVF3500, or
 
-7. Read and accept the Sensory license agreement.
-Wait for the script to complete the installation. This can take some time.
+   ```./auto_install.sh xvf3100```
 
-8. As a final step, the script will open http://localhost:3000 in a browser on the Raspberry Pi. Enter your Amazon Developer credentials and close the browser window when prompted. (You won't have to do this if you already have a valid configuration file.)  
-If you see a `400 Bad Request - HTTP` error it may be necessary to add http:// URLs to your device origins and return fields. To do this, go to your amazon developer AVS home page (https://developer.amazon.com/avs/home.html) and from `My products` select `manage` and then `security profile`. Now add http://localhost:3000 to the `Allowed origins` and http://localhost:3000/authresponse to the `Allowed return URLs`. Now refresh the browser window with the original error.
+   for the xCORE VocalFusion XVF3100.
```

Comment:
I am not sure if it is possible, but maybe these options can be written as a list.

Reply:
Many things are possible ;-) let me know if this meets your requirements...